tests/integration/targets/ndo_ptp_policy/tasks/main.yml

```diff
@@ -0,0 +1,233 @@
+          - nm_add_ptp_policy_again.previous.global.fabAnnounceTimeout == nm_add_ptp_policy_again.current.global.fabAnnounceTimeout == 3
+          - nm_add_ptp_policy_again.previous.global.fabAnnounceIntvl == nm_add_ptp_policy_again.current.global.fabAnnounceIntvl == 1
+          - nm_add_ptp_policy_again.previous.global.fabProfileTemplate == nm_add_ptp_policy_again.current.global.fabProfileTemplate == "aes67"
+          - nm_add_ptp_policy_again.previous.uuid is defined
+          - nm_add_ptp_policy_again.current.uuid is defined
+
+    # UPDATE
+    - name: Update a PTP policy (check mode)
+      cisco.mso.ndo_ptp_policy: &update_ptp_policy
+        <<: *add_ptp_policy
+        description: changed_description
+        admin_state: disabled
+        fabric_sync_interval: -2
+        global_domain: 1
+        global_priority1: 100
+        global_priority2: 200
+        state: present
+      check_mode: true
+      register: cm_update_ptp_policy
+
+    - name: Update a PTP policy
+      cisco.mso.ndo_ptp_policy:
+        <<: *update_ptp_policy
+      register: nm_update_ptp_policy
+
+    - name: Update a PTP policy again
+      cisco.mso.ndo_ptp_policy:
+        <<: *update_ptp_policy
+      register: nm_update_ptp_policy_again
+
+    - name: Assert PTP policy was updated
+      assert:
+        that:
+          - cm_update_ptp_policy is changed
+          - cm_update_ptp_policy.previous.description == ""
+          - cm_update_ptp_policy.previous.global.adminState == "enabled"
+          - cm_update_ptp_policy.previous.global.fabSyncIntvl == -3
+          - cm_update_ptp_policy.current == cm_update_ptp_policy.proposed
+          - cm_update_ptp_policy.current.description == cm_update_ptp_policy.proposed.description == "changed_description"
+          - cm_update_ptp_policy.current.global.adminState == cm_update_ptp_policy.proposed.global.adminState == "disabled"
+          - cm_update_ptp_policy.current.global.fabSyncIntvl == cm_update_ptp_policy.proposed.global.fabSyncIntvl == -2
+          - cm_update_ptp_policy.current.global.prio1 == cm_update_ptp_policy.proposed.global.prio1 == 100
+          - cm_update_ptp_policy.current.global.prio2 == cm_update_ptp_policy.proposed.global.prio2 == 200
+          - cm_update_ptp_policy.current.global.globalDomain == cm_update_ptp_policy.proposed.global.globalDomain == 1
+          - nm_update_ptp_policy is changed
+          - nm_update_ptp_policy.current.description == "changed_description"
+          - nm_update_ptp_policy.current.global.adminState == "disabled"
+          - nm_update_ptp_policy.current.global.fabSyncIntvl == -2
+          - nm_update_ptp_policy.current.global.globalDomain == 1
+          - nm_update_ptp_policy.current.global.prio1 == 100
+          - nm_update_ptp_policy.current.global.prio2 == 200
+          - nm_update_ptp_policy_again is not changed
+
+    # QUERY
+    - name: Update PTP policy name with UUID
```

Comment:
Should the update with UUID task be in the update test section above and also assert?